__tests__/new.ts

```diff
@@ -212,6 +212,16 @@ export class TypedArticleResource extends CoolerArticleResource {
 }
 
 export class FutureArticleResource extends CoolerArticleResource {
+  static url(id: any): string {
+    if (this.pk({ id }) !== undefined) {
+      if (this.urlRoot.endsWith('/')) {
+        return `${this.urlRoot}${this.pk({ id })}`;
+      }
+      return `${this.urlRoot}/${this.pk({ id })}`;
```

Comment:
nit:
```suggestion
      return `${this.urlRoot.replace(/\/$/, '/')}/${this.pk({ id })}`;
```

Reply:
It was suppsoed to be `${this.urlRoot.replace(/\/$/, '')}/${this.pk({ id })}`; right? (replace / with empty string)

Reply:
oh whoops, yes :-)